v2/common/src/main/java/com/google/cloud/teleport/v2/utils/WriteToGCSUtility.java

```diff
@@ -61,4 +61,10 @@ public enum FileFormat {
     AVRO,
     PARQUET;
   }
+
+  /** Set Enum BigtableSchemaFormat for all supported schema formats. */
+  public enum BigtableSchemaFormat {
```

Comment:
Should we move this thing out? Looks like we're putting some specific use-case stuff into common code

Reply:
Yes, this is specific to our use-case of writing from bigtable cdc -> gcs.